sdk/program/src/sysvar/instructions.rs

```diff
@@ -302,10 +302,11 @@ mod tests {
             message::{Message as LegacyMessage, SanitizedMessage},
             pubkey::Pubkey,
         },
+        std::collections::HashSet,
     };
 
     fn new_sanitized_message(message: LegacyMessage) -> SanitizedMessage {
-        SanitizedMessage::try_from_legacy_message(message).unwrap()
+        SanitizedMessage::try_from_legacy_message(message, &HashSet::default()).unwrap()
```

Comment:
Is this just to avoid importing `ReservedAccountKeys`? You use `ReservedAccountKeys::empty_key_set()` everywhere else.

Reply:
Yeah can't import it from solana-program